src/features/auth/auth.controller.ts

```diff
@@ -9,6 +11,29 @@ export class AuthController {
   @UseGuards(LocalAuthGuard)
   @Post('login')
   async login(@Request() req) {
-    return this._authService.login(req.user);
+    return this._authService.login(req.body);
+  }
+
+  @Post('register')
+  async register(@Request() req) {
+    return this._authService.register(req.body);
   }
+
+  @Get('me')
+  async me(@Request() req) {
```

Comment:
What's the purpose of the `me` endpoint?

Reply:
just Experiment with the concept of endpoint
i will remove it